joomla/joomla-cms
# [4.0] Contact Category list limits

PR for #33021 

Hopefully this resolves the bug reported with #30813. 
#30904 tried to fix it by removing the feature introduced with #23710 completely which was wrong

To test

- Create at least 10 contacts in the same category
- Create a menu item to display all the contacts in that category
- in the menu options make sure that Display Select is set to show (List layouts tab)
- In the global configuration check the value for default list limit (its probably 20)
- In the front end go to the new contact list menu item
- Check that the list limit select is displayed
- Check that the value is the same as in global config
- Check that changing the value in the select works
- Go back to the admin and edit the menu item and find the # contacts to list at the bottom of the list layouts tab
- Change the value from Global to 5 and save
- Clear "cookies" !!!
- In the front end go to the updated contact list menu item
- Check that the value is the same as you just set in the menu item (eg 5)
- Check that changing the value in the select works
